.prout.json

```diff
@@ -0,0 +1,5 @@
+{
+  "checkpoints": {
+    "DEV": { "url": "http://wrangler-todo.herokuapp.com", "overdue": "10M" },
```

Comment:
Hi @bitscraps - you actually have a JSON formatting error here, the trailing comma on this line makes this invalid JSON.

Something I haven't been very clear about is whether people should use their own copy of Prout, or use the instance we have set up for the Guardian at https://prout-bot.herokuapp.com - I wasn't fully decided myself, but having thought about it a bit more, I think it would be better if you ran your own copy of Prout - fairly easy to do with a free Heroku account. Visit https://github.com/guardian/prout#run-your-own-instance-of-prout and hit the 'Deploy to Heroku' button.